bin/plugin/commands/get-plugin-version.js

```diff
@@ -38,52 +37,50 @@ function doRunGetPluginVersion( settings ) {
 		throw Error( 'A slug must be provided via the --slug (-s) argument.' );
 	}
 
-	const pluginsFile = path.join( '.', settings.pluginsJsonFile );
-
-	// Buffer contents of plugins JSON file.
-	let pluginsFileContent = '';
+	// Resolve the absolute path to the plugins.json file.
+	const pluginsFile = path.join(
+		__dirname,
+		'../../../' + settings.pluginsJsonFile
+	);
 
 	try {
-		pluginsFileContent = fs.readFileSync( pluginsFile, 'utf-8' );
-	} catch ( e ) {
-		throw Error( `Error reading file at "${ pluginsFile }": ${ e }` );
-	}
-
-	// Validate that the plugins JSON file contains content before proceeding.
-	if ( ! pluginsFileContent ) {
-		throw Error(
-			`Contents of file at "${ pluginsFile }" could not be read, or are empty.`
-		);
-	}
+		// Read the plugins.json file synchronously.
+		const { modules, plugins } = require( pluginsFile );
 
-	const pluginsConfig = JSON.parse( pluginsFileContent );
-
-	// Check for valid and not empty object resulting from plugins JSON file parse.
-	if (
-		'object' !== typeof pluginsConfig ||
-		0 === Object.keys( pluginsConfig ).length
-	) {
-		throw Error(
-			`File at "${ pluginsFile }" parsed, but detected empty/non valid JSON object.`
-		);
-	}
-
-	const plugins = pluginsConfig.modules;
-	if ( ! plugins ) {
-		throw Error(
-			`File at "${ pluginsFile }" parsed, but the modules are missing, or they are misspelled.`
-		);
-	}
+		// Validate that the modules object is not empty.
+		if ( modules || Object.keys( modules ).length !== 0 ) {
+			for ( const moduleDir in modules ) {
+				const pluginVersion = modules[ moduleDir ]?.version;
+				const pluginSlug = modules[ moduleDir ]?.slug;
+				if (
+					pluginVersion &&
+					pluginSlug &&
+					settings.slug === pluginSlug
+				) {
```

Comment:
Ditto with above. This can be simplified with a for/of loop. If either the version or slug are empty, it should throw an error because the JSON file is bad.